scripts/kind-e2e/e2e.sh

```diff
@@ -328,10 +367,13 @@ deploy_nginx_cluster3
 
 test_connection
 
-# run dataplane E2e tests between the two clusters
-../bin/subctl verify-connectivity ${PRJ_ROOT}/output/kind-config/dapper/kind-config-cluster2 \
-                                  ${PRJ_ROOT}/output/kind-config/dapper/kind-config-cluster3 \
-                                  --verbose
+# dataplane E2E need to be modified for globalnet
+if [[ $globalnet = false ]]; then
+    # run dataplane E2e tests between the two clusters
+    ../bin/subctl verify-connectivity ${PRJ_ROOT}/output/kind-config/dapper/kind-config-cluster2 \
+                                      ${PRJ_ROOT}/output/kind-config/dapper/kind-config-cluster3 \
+                                      --verbose
+fi
```

Comment:
this check is not required once my PR merges (but do we need an official SM release before we can consume it here?) - https://github.com/submariner-io/submariner/pull/363/files

Reply:
yes.